apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -475,6 +475,16 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): J
             const fieldName = t("myAccount:components.profile.fields." + schema.name.replace(".", "_"),
                 { defaultValue: schema.displayName }
             );
+            // Define the field placeholder for text fields.
+            let innerPlaceholder = t("myAccount:components.profile.forms.generic." +
+                "inputs.placeholder",
+                {
+                    fieldName: fieldName.toLowerCase()
+                } );
+            // Concatenate the date format for the birth date field.
+            if (schema.name === "dateOfBirth") {
```

Comment:
Shall we add the DOB schema name here https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/users/constants/user-management-constants.ts#L94, please?

Reply:
This has been done.